### PR TITLE
Editing Design Elements

### PR DESCRIPTION
**Pull request by crmclean**
I changed a few elements within the main page related to design.

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.  

---

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ layout: workshop      # DON'T CHANGE THIS.
 # online workshop) are available at
 # https://carpentries.github.io/workshop-template/customization/index.html
 venue: "Woods Hole Oceanographic Institution"        # brief name of the institution that hosts the workshop without address (e.g., "Euphoric State University")
-address: "online"      # full street address of workshop (e.g., "Room A, 123 Forth Street, Blimingen, Euphoria"), videoconferencing URL, or 'online'
```
Comment (blongworth) [inline index.md]: "online" is picked up by the conditional location includes further down. I think it's case sensitive.

File: _includes/swc/schedule.html [hunk 2]
```diff
@@ -28,7 +28,7 @@ <h3>Day 3 Programming</h3>
       <tr> <td>15:00</td>  <td>Functions</td> </tr>
       <tr> <td>16:00</td>  <td>END</td> </tr>
     </table>
-    <h3>Day 4 Working with data</h3>
```
Comment (blongworth) [inline _includes/swc/schedule.html:31]: "with" is lowercase in title case.

Comment (crmclean) [reply _includes/swc/schedule.html:31]: done

**Review by crmclean — COMMENTED** [review]
Just finished updating everything!

**Pull request merged**